extensions/amp-lightbox-gallery/0.1/service/lightbox-manager-impl.js

```diff
@@ -41,13 +41,13 @@ import {toArray} from '../../../../src/core/types/array';
 const LIGHTBOX_ELIGIBLE_TAGS = new Set(['AMP-IMG', 'IMG']);
 
 // eslint-disable-next-line local/no-export-side-effect
-export const ELIGIBLE_TAP_TAGS = new Set(['AMP-IMG']);
+export const ELIGIBLE_TAP_TAGS = new Set(['AMP-IMG', 'IMG']);
```

Comment:
Since this is pattern we're embracing, it might be worth filing an FR to update our custom `no-export-side-effect` rule to allowlist Set/Map